docs/index.rst

```diff
@@ -15,3 +15,6 @@ The documentation for this package is here:
 
   ccdproc/install.rst
   ccdproc/index.rst
+..toctree::
+  :maxdepth: 1
+  changelog
```

Comment:
Need a blank line between the line with `maxdepth` and the line with `changelog`.